Sources/UberCore/Authentication/Tokens/AccessToken.swift

```diff
@@ -22,127 +22,45 @@
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
 
+
 import Foundation
 
-/**
- Stores information about an access token used for authorizing requests.
- 
- This class implements NSCoding, but its representation is an internal representation
- not compatible with the OAuth representation. Use an initializer if you want to serialize this
- via an OAuth representation.
- */
-public class AccessToken: NSObject, NSCoding {
-    /// String containing the bearer token.
-    public private(set) var tokenString: String
+public struct AccessToken: Codable, Equatable {
     
-    /// String containing the refresh token.
-    public private(set) var refreshToken: String?
+    public let accessToken: String?
```

Comment:
Seems a bit odd having a `accessToken` property within a `AccessToken` type. Should we rename `AccessToken` maybe to `OAuthTokenInfo`?

Reply:
I agree. I ended up keeping the AccessToken class name and changed the property to `tokenString`, as it was before.